### ISSUE TITLE
Missing Files

### ISSUE THREAD

**Issue by ankitatiisc** [chaneyddtt/Coarse-to-fine-3D-Animal#2]
Hi
I can't find files in the project like test_stanford_StanfordExtra_v12.npy, train_animal_pose.npy, train_stanford_StanfordExtra_v12.npy,breeds.csv. 
I downloaded Stanford dataset but that doesn't have Stanford extra dataset.

**Comment by chaneyddtt** [comment]
Hi @bogembo, you can download these annotations [here](https://github.com/benjiebob/StanfordExtra).

**Comment by ankitatiisc** [comment]
What about the animal pose dataset ? I am not able to find train_animal_pose.npy

**Comment by chaneyddtt** [comment]
Please refer to the instructions on how to download the dataset.

**Comment by ankitatiisc** [comment]
I followed the instructions. I could only find this file : "test_animal_pose.npy". The link I followed is : [.](https://github.com/benjiebob/WLDO/tree/master/data/animal_pose) File with training indices was missing. 
But I found a workaround. I created a training indices file by going through indices of the original animal_pose data. I filtered out the indices with dog category and then selected indices which does not belong to the test indices.
